fdm-app/app/components/blocks/fertilizer-applications/form.tsx

```
@@ -165,11 +179,43 @@ export function FertilizerApplicationForm<T extends typeof FormSchemaPartial>({
     ])
 
     useEffect(() => {
```

Biome (2.4.10)

[error] This hook specifies more dependencies than necessary: p_id.

(lint/correctness/useExhaustiveDependencies)